src/backend/base/langflow/base/mcp/util.py

```
@@ -395,7 +395,9 @@
             if not parsed.scheme or not parsed.netloc:
                 return False, "Invalid URL format. Must include scheme (http/https) and host."
 
-            async with httpx.AsyncClient() as client:
+            ssl_verify = server_config.get("ssl_verify", True)
```

Ruff (0.11.9)

Undefined name `server_config`

(F821)